Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM ruby:2.5.3
 
 WORKDIR /opt/puppet
+ENV LC_ALL=C.UTF-8
```

Comment:
Is this needed or just a byproduct of something you added for testing locally? Not sure if this file comes from modulesync

Reply:
Apparently the default locale is set to "POSIX" and when I run the test through docker container rake fails with "ArgumentError: invalid byte sequence in US-ASCII". I agree that this might be a bit offtopic for this issue, but fact is that without that line `docker build .` fails. Of course if I installed and ran everything locally stuff still passes.

Not sure what should be done here. Perhaps I can squash my two commits that deal with this bug and then create a separate commit / pull request for the issue that test failure when using docker. Perhaps there is even some nicer way (like that it is expected to pass some -e LC_ALL.. flag, but didn't find any documentation about it).